phenopacket-tools-builder/src/main/java/org/phenopackets/phenopackettools/builder/builders/PhenotypicFeatureBuilder.java

```diff
@@ -47,7 +47,7 @@ public PhenotypicFeatureBuilder onset(TimeElement time) {
     /**
      * @param isoISO8601 A string such as P10Y4M2D representing the age of onset/observation
      */
-    public PhenotypicFeatureBuilder isoISO8601onset(String isoISO8601) {
+    public PhenotypicFeatureBuilder iso8601onset(String isoISO8601) {
```

Comment:
@pnrobinson the argument name should be updated as well.

Reply:
ups, of course, done!